### DISCUSSION TITLE
Why is request.fspath deprecated in the latest branch ?

### DISCUSSION THREAD

**Discussion by hl037** [pytest-dev/pytest#8418] [Q&A] [answered]
From here : https://docs.pytest.org/en/latest/reference.html#request
I can see request.fspath will be deprecated. I can't find information about this decision and I was wondering what could I use instead to get the current test path ?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
Good catch, the path alias needs to be added there, `request.node.path` will do meanwhile

**Reply by nicoddemus** [reply]
@hl037 we have a short notice here: https://docs.pytest.org/en/latest/deprecations.html#node-fspath-in-favor-of-pathlib-and-node-path

However I wonder if we should expand that? We only mention some kind of reasoning briefly:

> As pytest tries to move off py.path.local we ported most of the node internals to pathlib.

Should we expand/create a new section with the rationale and future plans?

**Reply by RonnyPfannschmidt** [reply]
We should
